src/App.tsx

```diff
@@ -1,9 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  interface Move {
```

Comment:
i think this may be a typo of Movie kkkkkkkk

Reply:
yeah kkk Not MOVE, sorry